Marlin/src/feature/pause.cpp

```diff
@@ -107,25 +108,29 @@ void do_pause_e_move(const float &length, const float fr) {
 // public:
 
 bool move_away_flag = false;
+#if ENABLED(PAUSE_SPOOL_SWAP)
+bool swap_spool_enabled = false;
+int16_t swap_spool_stop=0;
+#endif
```

Comment:
Forced to make a 16 bits integer just to be compatible with lcd menu , but if you know how make a casting conversion , you can use a 8 bits allocation
I'm too much beginner , and i prefere make time to make the code work ,  and only after it works , i try to solve this type of less knowlege of coding loll